### ISSUE TITLE
Add "Breed status" classes

### ISSUE THREAD

**Issue by sabrinatoro** [monarch-initiative/vertebrate-breed-ontology#44]
Breed status should be added in VBO as classes. 
The following classes should be created

VBO id | Breed name | Parent ID NCBITaxon: | Definition
-- | -- | -- | --
VBO:0300000 | breed status |   |  
VBO:0300001 | breed recognition status | VBO:0300000 | Acknowledgment, by an international breed organization, of a group of animal as a breed.
VBO:0300002 | fully recognized | VBO:0300001 | Breed recognition status referring to a breed recognized by an international organization. This status also refers to breeds that are eligible to compete in championship classes.
VBO:0300003 | partially recognized | VBO:0300001 | Breed recognition status refering to a breed that is in the process of becoming fully recognized by an international organization. This status is given to breeds which are preliminarily or provisionally recognized.
VBO:0300004 | not_recognized | VBO:0300001 | Breed recognition status referring to a breed that is reported by an international organization but not recognized.
VBO:0300005 | domestication status | VBO:0300000 |  
VBO:0300006 | feral | VBO:0300005 | Domestication status that...
VBO:0300007 | domestic | VBO:0300005 | Domestication status that...
VBO:0300008 | wild | VBO:0300005 | Domestication status that...
VBO:0300009 | extinction status | VBO:0300000 |  
VBO:0300010 | not at risk of extinction | VBO:0300009 | extinction status that...
VBO:0300011 | a risk of extinction | VBO:0300009 | extinction status that...
VBO:0300012 | extinct | VBO:0300009 | extinction status that...

**Comment by sabrinatoro** [comment]
@katiermullen and I have added definitions for the breed recognition status. 

@franknic and @ImkeTammen 
Could you please write definitions for the following status? Thank you! 
- domestication status
  - feral
  - domestic
  - wild
- extinction status
  - not at risk of extinction
  - at risk of extinction
  - extinct

**Comment by franknic** [comment]
Thank you, Sabrina
Breed status is a very challenging issue! Many of the breeds in VBO are recognised only within a single country. And yet it is still quite valid for VBO to call them breeds. For domestication status, I am not aware that anyone has come up with useful definitions of those three categories. My only thought is that we could almost say that any group of animals that is regarded as a breed could be classified as "domestic". That would be a safe default assumption. For extinction status, because we are dealing almost solely with domesticated species, we are probably better to look to an authority such as DADIS rather than, for example, the IUCN Red List (https://portals.iucn.org/library/node/10315). DADIS (https://360.articulate.com/review/content/f940acc0-29db-4431-a37a-870c91e17b77/review) has 9 categories: Not at risk, Vulnerable, Endangered Maintained, Endangered, Critical  Maintained, Critical, Cryoconserved only, Extinct, and Unknown. The beauty of using DADIS categories is that we should be able to automatically include the DADIS category for each of the DADIS breeds. If we use different categories, we will then have the massive task of manually allocating a category to each breed. Also, the main reason for the creation of DADIS back in the 1990s was to identify breeds at risk of extinction throughout the world. Consequently, the DADIS people have given much thought to the issue of extinction status over several decades. I shall give this some more thought over the weekend.

**Comment by sabrinatoro** [comment]
@franknic: the domestication status comes from DAD-IS, and we currently have feral, domestic, and wild. 
Maybe we should reach out to them and ask for more information. 

The extinction status also comes from DAD-IS. I just saw 3, but I am happy to add all the categories if we need them. We would also need some definition for them though. We don't have this information in VBO yet, but they can easily be added once we have the data.

**Comment by franknic** [comment]
Thank you, Sabrina. I've had another look at the DADIS Excel sheet, and can see that they have 5 columns (DF to DJ) for extinction risk. One of these (Column DH = Detailed local risk status) includes all nine of their categories. In contrast, there is just the one column for domestication status with, as you say, those three categories. Extinction status is actually calculated by an algorithm, and the (very complex!) criteria are explained at https://360.articulate.com/review/content/f940acc0-29db-4431-a37a-870c91e17b77/review. 
Your suggestion is a good one: I will contact Gregoire and Roswitha.

**Comment by franknic** [comment]
After sending my previous message, I realised that most people have not seen the complete DADIS download file to which I was referring. The DADIS files I provided in the past, and which have been the basis for VBO, have contained only the most important columns, relating to our main aim of creating a computable, standardized list of breed names and synonyms in an informative hierarchy. DADIS have recently simplified the export of all their information, and an example of the entire DADIS file (downloaded on 4th May 2022) is at:
https://docs.google.com/spreadsheets/d/1ZDi8n6MhbWagzZddf89DoGyh5hBOeCma/edit#gid=1617249915. It is this file to which I was referring yesterday, in relation to the columns providing information on domestication and extinction status. As you will see, the entire file contains a wealth of information. But I am not advocating that we incorporate it all at this stage. For the first workable version of VBO, we should aim to keep it as simple as possible, concentrating on breed names, breed-country (for DADIS species) and synonyms.
